src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts

```diff
@@ -381,7 +378,7 @@ describe('EntryFieldsComponent', () => {
 
     actionSubject.next(action);
 
-    expect(store.dispatch).toHaveBeenCalledTimes(0);
+    expect(store.dispatch).toHaveBeenCalledTimes(3);
```

Comment:
Sorry, I'm not used to adding comments to the functionalities, but, it's the same explanation above, also change the name so that it is better understood.

Reply:
Cool!